ansible/roles/test/vars/testcases.yml

```diff
@@ -215,52 +215,52 @@ testcases:
     port_toggle:
       filename: port_toggle.yml
       vtestbed_compatible: no
-      topologies: [t0, t0-64, t0-64-32, t0-116, t1, t1-lag, t1-64-lag, t1-64-lag-clet, ptf32, ptf64]
+      topologies: [t0, t0-64, t0-64-32, t0-116, t1, t1-lag, t1-64-lag, ptf32, ptf64]
+
+    qos_sai_mlnx:
```

Comment:
Please also remove this ```qos``` section. Looks like it was pushed by mistake since it is not related to this PR.